src/components/BRadio/BRadio.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="container" :class="{ disabled: disabled }">
+    <div role="radiogroup" :aria-labelledby="groupId">
+      <label :id="groupId" class="description">
+      {{ groupLabel }}<span v-if="required" aria-hidden="true">*</span>
+      </label>
+      <div class="button-group" role="radiogroup">
+        <div v-for="(radio, index) in radios" class="button" :key="index">
+          <input type="radio" 
+            v-model="publicValue" 
+            v-validate.initial="index === 0 ? validations : {}"
+            v-bind="index === 0 ? validationAttributes : {}"
+            :aria-checked="radio.checked"
+            :aria-describedby="errors.any() ? `error-${groupId}` : ''"
+            :aria-labelledby="`${groupId} label-${radio.id}`"
+            :aria-posinset="index"
+            :aria-setsize="radios.length"
+            :disabled="radio.disabled || disabled"
+            :id="radio.id"
+            :name="groupId"
+            :value="radio.value"
+            @change="setCheckedRadio(radio)" 
+          />
+          <label :id="`label-${radio.id}`" :for="radio.id">
+            {{ radio.value }}
+          </label>
+        </div>
+        <div v-if="errors.any() && showErrors" :id="`error-${groupId}`" class="error">
+          <span v-for="(error,index) in errors.all()" :key="index">
+            {{ error }}
+          </span>
+        </div>    
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import validationMixIn from '../../mixins/validation'
+
+export default {
+  name: 'b-radio',
+  mixins: [validationMixIn()],
+  props: {
+    //Required props
+    /**
+    * Each radio group needs a unique id.
+    */
+    groupId: {
+      type: String,
+      required: true
+    },
+    /**
+    * Descriptive text about the radio group.
+    */
+    groupLabel: {
+      type: String,
+      required: true
+    },
+    /**
+    * Object containing all buttons included in radio group.
```

Comment:
Should this be an `Array of Objects containing...`?

Reply:
Sure.